lightwallet-client-lib/src/main/proto/compact_formats.proto

```diff
@@ -1,7 +1,11 @@
-syntax = "proto3";
+// Copyright (c) 2019-2020 The Zcash developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or https://www.opensource.org/licenses/mit-license.php .
+
+    syntax = "proto3";
 package cash.z.wallet.sdk.rpc;
 option java_package = "cash.z.wallet.sdk.internal.rpc";
-option go_package = "walletrpc";
+option go_package = "lightwalletd/walletrpc";
```

Comment:
is this some change you introduced or are these proto files coming from lightwalletd?

Reply:
Nope, I just downloaded the updated proto files from [lightwalletd repo](https://github.com/zcash/lightwalletd/tree/master/walletrpc) and added only one thing in each file and it's `java_package` option, which sets our local package for generated files.